Makefile

```diff
@@ -0,0 +1,53 @@
+PELICANOPTS=
+
+BASEDIR=$(CURDIR)
+INPUTDIR=$(BASEDIR)/content
+OUTPUTDIR=$(BASEDIR)/output
```

Comment:
I wonder how this will cope with github pages, I think they expect static html contents to live at the root of the repository but I may be wrong there; though we need to find a way to tell them that the static contents to serve lives within `output`.